<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <property name="horizontalSpacing">
     <number>8</number>
    </property>
    <property name="verticalSpacing">
     <number>23</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="Nombre">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="campoNombre"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="iDLabel">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="campoID"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="edadLabel">
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="campoEdad"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="bPMLabel">
      <property name="text">
       <string>BPM</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="campoBPM"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="spO2Label">
      <property name="text">
       <string>SpO2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="campoSpO2"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="temperaturaLabel">
      <property name="text">
       <string>Temperatura</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="campoTemp"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="botonGuardar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Formulario de registro</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
